app/config/config.yml

```diff
@@ -113,5 +117,5 @@ knp_snappy:
     pdf:
         enabled:    true
         # Edit this to change the location of wkhtmltopdf binary.
-        #binary:     /usr/local/bin/wkhtmltopdf #"\"C:\\Program Files\\wkhtmltopdf\\bin\\wkhtmltopdf.exe\"" for Windows users
```

Comment:
Please undo these changes before i can merge the PR :)

Reply:
ok no probleme.